Solution1/Solution1.Tests/ConsoleApp/Extensions/HttpStatusCodeExtensionsTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ConsoleApp.Extensions;
+using System.Net;
+using Xunit;
+
+namespace Weather.Tests.ConsoleApp.Extensions
+{
+    
+    public class HttpStatusCodeExtensionsTests
+    {
+        [Fact]
+        public void GetStringRepresentation_GetStringRepresentationFromHttpStatusCode_Success()
+        {
+            // Arrange
+            var httpStatusCode = HttpStatusCode.BadRequest;
+
+            // Act
+            var result = httpStatusCode.GetStringRepresentation();
+            
+            // Assert
+            var expected = "400 BadRequest.";
```

Comment:
seems unlogical that dot is here. what if you don't need to put dot next time in your output

Reply:
+